File: web3rpc/rpc-specs/code-samples/python/governance/itemCacheFromDb.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockNumber = 0

w3 = Web3(Web3.HTTPProvider(host))
governance_response = w3.governance.item_cache_from_db(blockNumber)

print(governance_response)

File: web3rpc/rpc-specs/code-samples/javascript/governance/itemCacheFromDb.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const blockNumber = 0;

  provider.governance
    .itemCacheFromDb(blockNumber, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
